cvat-core/src/annotations-objects.ts

```diff
@@ -394,15 +393,17 @@ class Annotation {
 }
 
 class Drawn extends Annotation {
-    protected frameMeta: AnnotationInjection['frameMeta'];
+    protected isFrameDeleted: (frame: number) => boolean;
```

Comment:
I undestand that here we are trying to avoid storing some kind of changing information inside of `framesInfo`, but from the logical perspective `isFrameDeleted` is not a property of an drawn annotation, it is a frame meta information. Maybe it would be better to place this callback into framesInfo object?

Reply:
Applied